augur/curate/titlecase.py

```diff
@@ -105,8 +90,7 @@ def run(args, records):
                 if failure_reporting is DataErrorMethod.WARN:
                     print_err(f"WARNING: {failure_message}")
     
-                    # Keep track of failures for final summary
-                    failures.append((record_id, field, date_string))
+                    failures.append((record_id, field, record.get(field, "")))
```

Comment:
This copy/paste bug fix should be part of the previous commit rather than an unrelated fixup here.

Reply:
Fixed up, thanks! I am starting to enjoy `fixup` and `rebase`